test/tests/interfacekernels/InterfaceSorption/tests

```diff
@@ -57,9 +55,8 @@
                 Materials/properties_2/type=ADGenericConstantMaterial
                 Postprocessors/flux_inner/type=ADSideDiffusiveFluxIntegral
                 Postprocessors/flux_outer/type=ADSideDiffusiveFluxIntegral
-                Outputs/file_base=interface_sorption_Sievert_out'
-    exodiff = 'interface_sorption_Sievert_out.e'
-    prereq = Sievert_ad
```

Comment:
I'm removing the pre-reqs because the previous tests are actually not required to run first. Now that each test has it's own input name and gold file (with aliases for tests that should predict the same behavior like ad/non-ad), there is no need for these tests to be run sequentially. 
I think this makes testing less constrained and makes debugging/updating easier, because otherwise many outputs had the same name.

Reply:
Yeah, distinguishing the outputs makes things a bit more robust, so that's a good change, but I also use pre-reqs to make testing more efficient. I mean, if something failed the the AD test, non-AD is much more unlikely to pass, etc. - then the later test doesn't need to take up resources to run. 

It's not a gigantic deal right now, so I don't necessarily require bringing it back, but with more tests, I look for ways to skip in logical progression when failures occur. Just wanted to get your reasoning.

Reply:
That's a fair point, let's discuss in person to pick a general guideline for TMAP8 cases. 

I personally like to have all the failures at once (especially on CI) to be able to debug effectively without launching the tests after each fix and realizing there were more issues that I didn't see because tests were skipped.